src/main/java/frc/robot/subsystems/intake/IntakeSubsystem.java

```diff
@@ -65,4 +52,50 @@ public void disable() {
         // TODO Auto-generated method stub
         
     }
+
+
+    public void spinForward() {
+        intake.set(ControlMode.PercentOutput, 0.5);
+    }
+
+
+    public void spinBackward() {
+        intake.set(ControlMode.PercentOutput, -0.5);
+    }
+
+
+    public void startSpinning() {
+        intake.set(ControlMode.PercentOutput, 0.5);
+    }
+
+
+    public void stopSpinning() {
+        intake.set(ControlMode.PercentOutput, 0);
+    }
+
+    public void pneumatics() {
+
+    //creates closed loop control to allow pressure to be stored
+        Compressor c = new Compressor(0);
+        c.setClosedLoopControl(true);
+        c.setClosedLoopControl(false);
+
+        boolean enabled = c.enabled();
+        boolean pressureSwitch =c.getPressureSwitchValue();
+        double current = c.getCompressorCurrent();
+
+    //declaring a solenoid
+        Solenoid exampleSolenoid = new Solenoid(1);
+        exampleSolenoid.set(true);
+        exampleSolenoid.set(false);
```

Comment:
We shouldn't need any compressor or single solenoid code, just a doublesolenoid that toggles between forwards and reverse each time the button is pressed.